
int32 block_number
int32 block_zone
---
bool success
